vaadinfrontend/src/main/java/life/qbic/datamanager/views/support/experiment/ExperimentItem.java

```diff
@@ -0,0 +1,132 @@
+package life.qbic.datamanager.views.support.experiment;
+
+import com.vaadin.flow.component.ComponentEventListener;
+import com.vaadin.flow.component.html.Div;
+import com.vaadin.flow.component.html.Span;
+import com.vaadin.flow.component.icon.Icon;
+import com.vaadin.flow.component.icon.VaadinIcon;
+import com.vaadin.flow.theme.lumo.LumoUtility.BorderRadius;
+import java.io.Serial;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import life.qbic.datamanager.views.general.Card;
+import life.qbic.projectmanagement.domain.project.experiment.Experiment;
+import life.qbic.projectmanagement.domain.project.experiment.ExperimentId;
+
+/**
+ * <b><class short description - 1 Line!></b>
+ *
+ * <p><More detailed description - When to use, what it solves, etc.></p>
+ *
+ * @since <version tag>
+ */
+public class ExperimentItem extends Card {
```

Comment:
NIT: You could think about moving the ExperimentItem within the ExperimentItemCollection class since it will solely exist in that context. 
One possiblity could be to provide a method in the `ExperimentItemCollection` Div that creates the `ExperimentItem` for an added Experiment. 
E.G.
`experimentItemCollection.addItemForExperiment(Experiment experiment)`

Reply:
The item collection is just a container, for readability I moved the item in an own class. Since it is in the same package, i dont see the issue here